README.md

```diff
@@ -10,7 +10,6 @@ WallGo is open-source scientific software for computing the bubble wall speed fo
 
 **Version** v0.1.0 (alpha)
 
-![WallGo tests](https://github.com/Wall-Go/WallGo/actions/workflows/main.yml/badge.svg)
```

Comment:
I guess we don't want to keep this modification - this was just because it didn't work on bitbucket